test/AWS.Logger.AspNetCore.Tests/TestClass.cs

```diff
@@ -63,14 +63,14 @@ public ILoggerTestClass(TestFixture testFixture) : base(testFixture)
             }
 
             var loggingFactoryService = this.ServiceCollection.FirstOrDefault(x => x.ServiceType is ILoggerFactory);
-            this.Provider = this.ServiceCollection.AddLogging()
+            this.Provider = this.ServiceCollection.AddLogging(logging => logging.SetMinimumLevel(LogLevel.Debug))
                                     .BuildServiceProvider();
 
             if (loggingFactoryService == null)
             {
                 var loggingFactory = this.Provider.GetService<ILoggerFactory>();
-                loggingFactory.AddAWSProvider(ConfigSection);
-                Logger = loggingFactory.CreateLogger<ILoggerTestClass>();
+                loggingFactory.AddAWSProvider(ConfigSection);                
```

Comment:
NIT extra spaces at the end of the line (3 times in this file)

Reply:
Fixed